src/foam/mlang/mlang.js

```diff
@@ -3956,6 +3956,74 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
```

Comment:
@kgrgreer going forward can we move new entries into their own file? Makes debugging in browser possible.

Reply:
Sure.